pyproject.toml

```diff
@@ -136,14 +136,17 @@ target-version = ['py38', 'py39', 'py310', 'py311']
 [tool.cibuildwheel]
 manylinux-x86_64-image = "manylinux2014"
 manylinux-i686-image = "manylinux2014"
-skip = "pp* cp36-* cp37-* *musllinux*"
+skip = "pp* cp36-* cp37-* *musllinux* *win32 *i686"
 test-skip = "*win32 *linux_i686"
 test-command = "python {project}/examples/python/stochastic_swap.py"
 # We need to use pre-built versions of Numpy and Scipy in the tests; they have a
 # tendency to crash if they're installed from source by `pip install`, and since
 # Numpy 1.22 there are no i686 wheels, so we force pip to use older ones without
 # restricting any dependencies that Numpy and Scipy might have.
 before-test = "pip install --only-binary=numpy,scipy numpy scipy"
+# Some jobs locally override the before-build and environment configuration if a
+# specific job override is needed. For example tier 1 platforms locally override
```

Comment:
Is the rest of this example sentence cut off?

Reply:
Lol, yeah my brain must have just turned off mid-sentence:

```suggestion
# specific job override is needed. For example tier 1 platforms locally override
# the before-build and environment configuration to enable PGO,
```